worldflags.txt

```diff
@@ -0,0 +1,196 @@
+What country is represented by this flag? https://i.imgur.com/V0y3m1h.png `Malawi
+What country is represented by this flag? https://i.imgur.com/nieDYXN.png `Malaysia
+What country is represented by this flag? https://i.imgur.com/1VsVFYK.png `Maldives
+What country is represented by this flag? https://i.imgur.com/re1GuyK.png `Mali
+What country is represented by this flag? https://i.imgur.com/BqwHKYv.png `Malta`Hvid Rød
+What country is represented by this flag? https://i.imgur.com/zBYYxpN.png `Marshall Islands
+What country is represented by this flag? https://i.imgur.com/1o8hBla.png `Mauritania
+What country is represented by this flag? https://i.imgur.com/bjtyCp5.png `Mauritius
+What country is represented by this flag? https://i.imgur.com/Os7kijz.png `Mexico
+What country is represented by this flag? https://i.imgur.com/b9STlvA.png `Micronesia
+What country is represented by this flag? https://i.imgur.com/j7Ig3Bu.png `Moldova
+What country is represented by this flag? https://i.imgur.com/oMLG5ri.png `Mongolia
+What country is represented by this flag? https://i.imgur.com/TPz5Gaj.png `Montenegro
+What country is represented by this flag? https://i.imgur.com/CXQK227.png `Morocco
+What country is represented by this flag? https://i.imgur.com/BgdKIb2.png `Mozambique
+What country is represented by this flag? https://i.imgur.com/00SGRuz.png `Myanmar`Burma
+What country is represented by this flag? https://i.imgur.com/VeiCyLB.png `Namibia
+What country is represented by this flag? https://i.imgur.com/GUUtDv6.png `Nauru
+What country is represented by this flag? https://i.imgur.com/mAgkfeH.png `Nepal
+What country is represented by this flag? https://i.imgur.com/qNyJQ5d.png `The Netherlands`Netherlands`Holland
+What country is represented by this flag? https://i.imgur.com/WElCUGn.png `New Zealand
+What country is represented by this flag? https://i.imgur.com/dmlTGO4.png `Nicaragua
+What country is represented by this flag? https://i.imgur.com/DeINlqb.png `Niger
+What country is represented by this flag? https://i.imgur.com/hbJ4J2A.png `Nigeria
+What country is represented by this flag? https://i.imgur.com/8LFhQVn.png `North Korea`People's Republic of Korea
+What country is represented by this flag? https://i.imgur.com/i4srkYW.png `Norway
+What country is represented by this flag? https://i.imgur.com/ttIhE04.png `Oman
+What country is represented by this flag? https://i.imgur.com/QclX66O.png `Pakistan
+What country is represented by this flag? https://i.imgur.com/weIW0md.png `Palau`Palaos Bandera`Bandera de Palaos
+What country is represented by this flag? https://i.imgur.com/Vbr5Tam.png `Panama
+What country is represented by this flag? https://i.imgur.com/y5PJwXq.png `Papua New Guinea
+What country is represented by this flag? https://i.imgur.com/jKRkoe5.png `Paraguay
+What country is represented by this flag? https://i.imgur.com/CEetl8W.png `Peru
+What country is represented by this flag? https://i.imgur.com/3H3Lfat.png `Philippines
```

Comment:
also Republic of the Philippines?

Reply:
what do you think of this @MiniJennJenn?

Reply:
I already updated the PR with it as seen below.